automation_tools/__init__.py

```diff
@@ -428,8 +433,12 @@ def setup_abrt():
         '/etc/foreman-proxy/settings.d/abrt.yml')
 
     # run the required commands
-    run('systemctl start abrtd')
-    run('systemctl start abrt-ccpp')
+    if os_version >= 7:
+        run('systemctl start abrtd')
+        run('systemctl start abrt-ccpp')
```

Comment:
FYI, you can place all of the units being started on one line:

    systemctl start abrtd abrt-ccpp

I'm not terribly worried about making this change right now, though. As per a conversation with @cswiii, we may move SysV/systemd daemon management logic into a separate function. Something like this:

```python
def manage_daemon(action, daemon):
    if distro_info()[1] == 6:
        command = 'service {} {}'.format(daemon, action)
    else:
        command = 'systemctl {} {}'.format(action, daemon)
    return run(command)
```